### PR TITLE
DOCSP-37090 Advanced Kafka Settings (Reference)

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Additional Kafka Advanced Settings 

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-37090/installation/kafka-deployments/advanced-settings/

## JIRA
https://jira.mongodb.org/browse/DOCSP-37090

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6602f9d26fc5f04850177b0d

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/installation/kafka-deployments/advanced-settings.txt [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    You can add additonal remote Java Management Extensions (JMX) for SSL 
+    configuration by prefixing them with ``migrator.kafka.connect.metrics.jmx.properties``.
+
+   ``migrator.kafka.enable.topic.cleanup``
+    Default: false
+    
+    Cleans up the topics created during the last migration job.
+
+   ``migrator.connector.source.common.errors.max.retries``
+    Default: 5
```
Comment (mmaville-mdb) [inline source/installation/kafka-deployments/advanced-settings.txt:69]: Monospace default number value for consistency.

File: source/installation/kafka-deployments/advanced-settings.txt [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    does not exceed ``migrator.connector.source.common.errors.retry.delay.max.ms``.
+
+   ``migrator.connector.source.common.errors.retry.delay.max.ms``
+    Default: 60000 ms
+
+    For the source connector, this specifies the maximum delay in milliseconds 
+    between retries after a retriable failure.
+
+   ``migrator.connector.sink.common.errors.max.retries``
+    Default: 5
```
Comment (mmaville-mdb) [inline source/installation/kafka-deployments/advanced-settings.txt:89]: Monospace default number value for consistency.

**Review by mmaville-mdb — APPROVED** [review]
LGTM! I left some non-blocking suggestion for monospacing the default plain number values. I'm not sure if you want to look at the other default values, since it's a mix of boolean and numbers with time modifiers (ms). Trying to find guidance, but thought I'd bring it up.

File: source/installation/kafka-deployments/advanced-settings.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/installation/kafka-deployments/advanced-settings.txt]: Thanks for bringing this up! I am actually going to follow the server parameters page example and not use monospacing for the values and italicize the _Default_.

**Pull request merged**